pogom/utils.py

```diff
@@ -966,8 +957,9 @@ def is_imagemagick_binary(binary):
         process = subprocess.Popen([binary, '-version'],
                                    stdout=subprocess.PIPE)
         out, err = process.communicate()
-        return "ImageMagick" in out
-    except Exception:
+        return "ImageMagick" in out.decode('utf8')
+    except Exception as e:
+        print(e)
```

Comment:
This is printing a warning on every startup

Reply:
Please post that warning

Reply:
```
[Errno 2] No such file or directory: 'magick': 'magick'
```
The function is checking two binaries for imagemagick. Depending on your OS, you will always have this exception printed.

Reply:
Now I understand, what the problem was... fixed.